libnestutil/beta_normalization_factor.h

```diff
@@ -75,7 +75,7 @@ beta_normalization_factor( const double tau_rise, const double tau_decay )
 {
   const double tau_difference = tau_decay - tau_rise;
   double peak_value = 0;
-  double normalization_factor = 0;
+  double normalization_factor;
```

Comment:
There is no need to initialize the variable here, as the value is never used before it is set to another value in the following lines.

Reply:
That is true for the code as it is now, but it may change in the future. But initializing a factor with 0 seems dangerous, 1 might be a better value. But looking at the code of the entire function, I would suggest to remove the `normalization_factor` variable entirely and just return directly from the if-else below. Since that is the last statement of the function, I think that is sufficiently clean.

Reply:
Valid points! Cleaned it up.